ql/src/test/queries/clientpositive/antijoin2.q

```diff
@@ -0,0 +1,41 @@
+set hive.merge.nway.joins=false;
```

Comment:
Can we also add the explain cbo plan and explain plan here?

Reply:
I added explain and explain cbo for each of the queries.